ProgrammingFundamentals/Session1/Assignment2/Assignment2.java

```diff
@@ -0,0 +1,35 @@
+import java.util.*;
+/**
+ *
+ *
+ *	@author  Vinod Patel
+ *
+ */
+public class Assignment2 {
+	// Removing duplicate elements from Array
+	public int[] removeDuplicate(int input[]) {
+		ArrayList<Integer> al = new ArrayList<Integer>();//creating dynamic array to store non duplicate elements

```

Comment:
You can write this program without using of this array list. You can directly put non duplicate values in int[] res which you have done below in the code. Try to use less number of variables.